.github/workflows/health_checks.yml

```diff
@@ -95,10 +95,25 @@ jobs:
       - uses: ./.github/actions/setup_node
       - uses: ./.github/actions/restore_build_cache
       - run: cd packages/cli && npm link
-      - name: Configure AWS Credentials
+      - name: Configure test tooling credentials
+        id: credentials
         uses: aws-actions/configure-aws-credentials@04b98b3f9e85f563fb061be8751a0352327246b0 # version 3.0.1
         with:
-          role-to-assume: ${{ secrets.E2E_RUNNER_ROLE_ARN }}
+          role-to-assume: ${{ secrets.E2E_TOOLING_ROLE_ARN }}
```

Comment:
nit: since this step is getting bigger, should we move it to a re-usable action?